api/scpca_portal/models/sample.py

```diff
@@ -88,6 +88,24 @@ def get_output_single_cell_metadata_file_path(scpca_sample_id):
     def get_output_spatial_metadata_file_path(scpca_sample_id):
         return os.path.join(common.OUTPUT_DATA_DIR, f"{scpca_sample_id}_spatial_metadata.tsv")
 
+    @property
+    def modalities(self):
+        modalities = []
+
+        if self.has_bulk_rna_seq:
+            modalities.append("Bulk")
+
+        if self.has_cite_seq_data:
+            modalities.append("CITE-seq")
+
+        if self.has_spatial_data:
+            modalities.append("Spatial Transcriptomics")
+
+        if modalities:
+            return ", ".join(modalities)
+
+        return None
```

Comment:
```suggestion

```

Reply:
No return == return None

Reply:
yeah I tend to prefer explicit returns in languages that are less strict about that, but I'm happy to defer to your preference

Reply:
I see. For me it just looks cleaner this way.
Thank you!